### ISSUE TITLE
[BUG] Link to the registered model from the run's artifacts section does not honor static prefix

### ISSUE THREAD

**Issue by mgmachado** [mlflow/mlflow#2213]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in MLflow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 18.04.3 LTS
- **MLflow installed from (source or binary)**: pip install mlflow[extras]
- **MLflow version (run ``mlflow --version``)**: mlflow, version 1.4.0
- **Python version**: 3.7.3
- **npm version, if running the dev UI**: N/A
- **Exact command to reproduce**:

### Describe the problem
I started an MLFlow tracking server with a static prefix, logged runs against the tracking URI and registered a model successfully. When I go to the experiments page > click on a run > and attempt to click on the registered model, the link is broken because the static prefix is missing. Screenshot:
![MLFlow_static_prefix_error](https://user-images.githubusercontent.com/57337661/70948473-25e51080-2021-11ea-8583-9b46bdc1c0ac.png)

### Code to reproduce issue
1. Start mlflow tracking server using:
```
mlflow server --backend-store-uri postgresql://<user>:<password>@<ip>/<db> --default-artifact-root <path/to/a/local/dir> --host 0.0.0.0 --static-prefix /myprefix
```

2. Train model (steps from tutorial):
```
git clone https://github.com/mlflow/mlflow
cd examples
python sklearn_elasticnet_wine/train.py
```
with the following lines added to examples/sklearn_elasticnet_wine/train.py:
```
    remote_server_uri = "http://localhost:5000"
    mlflow.set_tracking_uri(remote_server_uri)
    mlflow.set_experiment("/my-experiment-at-subpath")
```
3. Go to http://localhost:5000/myprefix/#/,  navigate to the run and use the UI to register the model.

The link to the model is now broken because it does not include the static prefix (see screenshot).

**Comment by elbowz** [comment]
+1

**Comment by mgmachado** [comment]
This seems to be the issue:

In mlflow/server/js/src/model-registry/routes.js line 15 should be ```return `#${modelRoute}`;``` (i.e. without the leading slash)

**Comment by mparkhe** [comment]
Hi @Zangr: Can you please take this one?

**Comment by Zangr** [comment]
@mgmachado, thank you for reporting this, and the RCA LGTM as well!

**Comment by Zangr** [comment]
#2234 https://github.com/mlflow/mlflow/pull/2234

**Comment by pogil** [comment]
Fixed in #2234
